avm/res/network/load-balancer/inbound-nat-rule/main.bicep

```diff
@@ -8,15 +8,15 @@ param loadBalancerName string
 @description('Required. The name of the inbound NAT rule.')
 param name string
 
-@description('Required. The port for the external endpoint. Port numbers for each rule must be unique within the Load Balancer.')
-@minValue(1)
+@description('Conditional. The port for the external endpoint. Port numbers for each rule must be unique within the Load Balancer. Required if FrontendPortRangeStart and FrontendPortRangeEnd are not specified.')
+@minValue(-1)
 @maxValue(65534)
-param frontendPort int
+param frontendPort int = -1
```

Comment:
Hmm - but then something is still off with the implemention
```bicep
param frontendPort int = -1 
(...)
  frontendPort: frontendPort != -1 ? frontendPort : null
(...)
```
right?

If you cannot assign `null` to the `frontendPort`, then the else case may never happen, correct? Otherwise the following should be valid
```bicep
param frontendPort int?
(...)
  frontendPort: frontendPort
(...)
```

I must be missing something 😄 Would be great if we could have a chat if you have a minute some time (: